vm/actor/tests/common/mod.rs

```diff
@@ -125,6 +142,26 @@ impl<'a, BS: BlockStore> MockRuntime<'a, BS> {
         assert!(addr.len() > 0, "addrs must be non-empty");
         *self.expect_validate_caller_addr.borrow_mut() = Some(addr.to_vec());
     }
+    pub fn expect_validate_caller_type(&self, ids: &[Cid]) {
```

Comment:
can you add a dead code tag on this? not used for other tests and throwing warnings